coalahtml/coala_html.py

```diff
@@ -35,7 +35,7 @@ def main():
 
     dir_path = create_dir(os.path.abspath(args.dir))
 
-    if len(list(os.walk(dir_path))) > 0:
+    if len(list(os.walk(dir_path))) > 0:  # pragma: no cover
```

Comment:
I'll be happy with either decision. Evaluate all pros and cons and try to figure out the best trade off. There can be another solution to this. For example, asking user if we detect them some of the files are not what they should be if he wants to leave them as is, or update it. Maybe, make it a cli option for coala-html.